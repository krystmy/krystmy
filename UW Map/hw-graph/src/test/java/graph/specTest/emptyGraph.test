# Test creating an empty graph

# Create a graph
CreateGraph empty

# Print the nodes in the graph
ListNodes empty
