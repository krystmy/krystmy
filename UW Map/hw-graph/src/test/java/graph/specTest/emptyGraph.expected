# Test creating an empty graph

# Create a graph
created graph empty

# Print the nodes in the graph
empty contains: